jbpm-workitems/jbpm-workitems-rest/src/main/java/org/jbpm/process/workitem/rest/RESTWorkItemHandler.java

```diff
@@ -790,9 +797,10 @@ public void executeWorkItem(WorkItem workItem,
         if(headers == null) {
             headers = "";
         }
-
+    
         HttpClient httpClient = getHttpClient(readTimeout,
-                                              connectTimeout);
+                                              connectTimeout,
+                                              params);
```

Comment:
you could have passed the urlStr in a different parameter for clarity (instead of retrieving again)